### ISSUE TITLE
it seems an mistake in FactorEncoder(nn.Module)?

### ISSUE THREAD

**Issue by BigAndSweet** [x7jeon8gi/FactorVAE#3]
that is, self.softmax = nn.Softmax(dim=1) should be self.softmax = nn.Softmax(dim=0) based on eq(9) of the paper?

**Comment by x7jeon8gi** [comment]
Do you mean equation 6 in the paper?

**Comment by x7jeon8gi** [comment]
Do you mean equation 6 in the paper?

**Comment by x7jeon8gi** [comment]
I am truly grateful for your insight. You are absolutely correct, and I acknowledge that this oversight was mine. I will promptly rectify it by adjusting to dim=0. My sincere thanks for your invaluable assistance.
